pkg/server/server.go

```diff
@@ -387,7 +389,12 @@ func (s *ProxyServer) Proxy(stream client.ProxyService_ProxyServer) error {
 				return
 			}
 			if err != nil {
-				klog.ErrorS(err, "Stream read from frontend failure")
+				rpcCode := status.Convert(err).Code()
+				if rpcCode == codes.Canceled {
+					klog.V(2).InfoS("Stream has been canceled", "rpc_code", rpcCode)
```

Comment:
we know it's canceled, so maybe "rpc_code" is uncessary here (but could be helpful below in the ErrorS)

Reply:
Hi! Do you mean shall I remove the logging when it is canceled? Thanks!

Reply:
I think it was also add the rpcCode to the error message when rpcCode != codes.Cancelled.

Reply:
Thank you! It is clear now :)